### PR TITLE
Feature : CRUD User

### LINKED ISSUE #2: unable to submit form inspection
lorem ipsum

### PR DESCRIPTION
**Pull request by afiqaiman98**

### CODE REVIEW

File: routes/web.php [hunk 1]
```diff
@@ -13,12 +15,45 @@
 |
 */
 
-Route::get('/', function () {
-    return view('welcome');
-});
+// Route::get('/', function () {
```
Comment (ImranSasudin) [inline routes/web.php:18]: You can remove this if you don't want to use it

**Review by ImranSasudin — CHANGES_REQUESTED** [review]
Please correct this file

File: routes/web.php (hunk 1 above)
Comment (afiqaiman98) [reply routes/web.php:18]: Saye beki dh

**Review by ImranSasudin — APPROVED** [review]
LGTM!

**Pull request merged**